msvc-full-features/Cataclysm-vcpkg-static.sln

```diff
@@ -39,7 +39,6 @@ Global
 		{2C1ECEE1-9686-4C3C-8DE1-88996EE43378}.Debug|x86.ActiveCfg = Debug|Win32
 		{2C1ECEE1-9686-4C3C-8DE1-88996EE43378}.Debug|x86.Build.0 = Debug|Win32
 		{2C1ECEE1-9686-4C3C-8DE1-88996EE43378}.Release|x64.ActiveCfg = Release|x64
-		{2C1ECEE1-9686-4C3C-8DE1-88996EE43378}.Release|x64.Build.0 = Release|x64

```

Comment:
```suggestion
		{2C1ECEE1-9686-4C3C-8DE1-88996EE43378}.Release|x64.Build.0 = Release|x64
```

Reply:
Yeah those will be slipping though a bit occaionally. VS doesn't like something about tests and JSON formatter and fails to compile them completelyh.